src/modules/navigator/GeofenceBreachAvoidance/geofence_breach_avoidance.cpp

```diff
@@ -281,7 +296,7 @@ void GeofenceBreachAvoidance::updateMinVertDistToFenceMultirotor()
 
 Vector2d GeofenceBreachAvoidance::waypointFromHomeToTestPointAtDist(float distance)
 {
-	Vector2d test_point = getFenceViolationTestPoint();
+	Vector2d test_point = getFenceHorizontalLoiterPoint();
 	float bearing_home_current_pos = get_bearing_to_next_waypoint(_home_lat_lon(0), _home_lat_lon(1), test_point(0),
```

Comment:
This change simply results in Hold being derived from the braking distance instead of the drone's current position (as TestPoint is now based on drone position).